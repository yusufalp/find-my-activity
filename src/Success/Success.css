.success {
  margin-top: 120px;
}