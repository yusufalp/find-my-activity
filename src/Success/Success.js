import React from 'react';
import './Success.css'

export default function Success(){
  return(
    <div>
      <h2 className="success">Successful</h2>
      <p>Your submission is accepted</p>
      <img className="group-kids" src="/images/success-children.png" alt="children with a heart"></img>
    </div>
  )
}