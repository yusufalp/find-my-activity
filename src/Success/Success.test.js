import React from 'react';
import ReactDOM from 'react-dom';
import Success from './Success';
import renderer from 'react-test-renderer';

it('renders without crashing', () => {
  const div = document.createElement('div');
  ReactDOM.render(<Success />, div);
  ReactDOM.unmountComponentAtNode(div);
});

it('renders the UI as expected', () => {
  const tree = renderer
    .create(<Success />)
    .toJSON();
  expect(tree).toMatchSnapshot();
});