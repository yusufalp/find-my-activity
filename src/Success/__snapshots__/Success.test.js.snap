// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders the UI as expected 1`] = `
<div>
  <h2
    className="success"
  >
    Successful
  </h2>
  <p>
    Your submission is accepted
  </p>
  <img
    alt="children with a heart"
    className="group-kids"
    src="/images/success-children.png"
  />
</div>
`;
